shell/platform/glfw/flutter_glfw.cc

```diff
@@ -740,7 +740,7 @@ bool FlutterDesktopRunWindowEventLoopWithTimeout(
     FlutterDesktopWindowControllerRef controller,
     uint32_t timeout_milliseconds) {
   auto wait_duration = timeout_milliseconds == 0
```

Comment:
Let's stop using `auto` here since the right hand size is no longer consistent; I don't actually know offhand which way the conversion will go, so I no longer know the exact type. We should explicitly use nanoseconds.